.github/workflows/samples-dotnet.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-dotnet@v3.2.0
         with:
-          dotnet-version: '7.0.x'
+          dotnet-version: '8.0.x'
```

Comment:
perfectly fine with this change.

shall we add another workflow to test .net7 to ensure the samples (at least fake petstore) work wtih .net7?

Reply:
I don't think we need another workflow. If we want to test .net 7 then we would need to add samples and set the config to use it. Do we want every framework to have samples? I can't help but think it's a bit overkill. Microsoft is pretty good at keeping the code backwards compatible. But we can add more samples if desired.

Reply:
Not saying MS is doing a bad job. Previously we encountered an issue a PR included syntax only available in the latest .net framework and therefore breaking backward compatibility. Just want to avoid the same issue from happening again.

Reply:
Okay so we want more frameworks tested. I can add in another pr.

Reply:
I'll do it shortly. Don't worry.

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/pull/17098